src/celeritas/phys/PhysicsStepUtils.hh

```diff
@@ -194,9 +195,22 @@ calc_mean_energy_loss(ParticleTrackView const& particle,
     {
         auto grid_id = physics.value_grid(VGT::energy_loss, ppid);
         CELER_ASSERT(grid_id);
-        auto calc_eloss_rate
-            = physics.make_calculator<EnergyLossCalculator>(grid_id);
-        eloss = Energy{step * calc_eloss_rate(pre_step_energy)};
+
+        size_type order = physics.scalars().spline_eloss_order;
+        CELER_ASSERT(order > 0);
```

Comment:
Since we already asserted by construction that the physics is "true", let's skip this check.